client/my-sites/checkout/cart/cart-coupon.jsx

```diff
@@ -89,6 +89,7 @@ export class CartCoupon extends React.Component {
 			<form onSubmit={ this.applyCoupon } className={ 'cart__form' }>
 				<input
 					type="text"
+					id="coupon-code"
```

Comment:
can we please use a data attribute like `data-e2e-coupon-code` instead of an id? Using the data attribute makes it clear these are used by the e2e tests

Reply:
Right! I'll push this change in a bit